### PR TITLE
updated descriptions and input/output property to counter operation

### PR DESCRIPTION
**Pull request by soumyanayak71**
updated descriptions and input/output property to counter operation.

### CODE REVIEW

File: cs-hashicorp-terraform/src/main/java/io/cloudslang/content/hashicorp/terraform/utils/Descriptions.java [hunk 1]
```diff
@@ -237,6 +237,15 @@
     }
 
     public static class Counter{
-        public static final String FAILURE_MESSAGE="Something went wrong";
+        public static final String FAILURE_MESSAGE = "Something went wrong";
+        public static final String COUNTER_DESC = "Counts from one number to another number.";
+        public static final String RESULT_STRING_DESC = "he primary result is resultString, Result can also be used. result (All lower case) should not be used as it is the response code.";
```
Comment (IuliaMoldovan) [inline cs-hashicorp-terraform/src/main/java/io/cloudslang/content/hashicorp/terraform/utils/Descriptions.java]: it should be "the"

File: cs-hashicorp-terraform/src/main/java/io/cloudslang/content/hashicorp/terraform/utils/Descriptions.java [hunk 2]
```diff
@@ -237,6 +237,15 @@
...
+        public static final String RESULT_DESC = "If successful, returns the complete API response. In case of an error this output will contain the error message.";
+        public static final String FROM_DESC = "The number to start counting at.";
+        public static final String TO_DESC = "The number to count to.";
+        public static final String RESET_DESC = "If true, then the counter will restart counting from the beginning.";
+        public static final String INCREMENT_BY_DESC = "he number to increment by while counting. If unspecified this is 1. If you wanted to count 2,4,6,8 this would be 2.";
```
Comment (IuliaMoldovan) [inline cs-hashicorp-terraform/src/main/java/io/cloudslang/content/hashicorp/terraform/utils/Descriptions.java]: here also

File: cs-hashicorp-terraform/src/main/java/io/cloudslang/content/hashicorp/terraform/utils/Descriptions.java (hunk 1 above)
Comment (soumyanayak71) [reply cs-hashicorp-terraform/src/main/java/io/cloudslang/content/hashicorp/terraform/utils/Descriptions.java]: incorporate

File: cs-hashicorp-terraform/src/main/java/io/cloudslang/content/hashicorp/terraform/utils/Descriptions.java (hunk 2 above)
Comment (soumyanayak71) [reply cs-hashicorp-terraform/src/main/java/io/cloudslang/content/hashicorp/terraform/utils/Descriptions.java]: incorporated

**Pull request merged**